furniro-front-end/src/app/Commons/data/HomeData.tsx

```diff
@@ -0,0 +1,14 @@
+import HomeBackground from "../images/background.png";
+import { StaticImageData } from "next/image";
+
+interface imageProps {
+  src: string | StaticImageData;
+  alt: string;
+}
+
+export const backgroundData = () => {
+  {
+    src: HomeBackground;
+    alt: "Background";
+  }
+};
```

Comment:
You dont need to have a data for a simple image in a single component, just use it directly in the component and it doesn't have to have props for it